scripts/src/public/services/orders.ts

```diff
@@ -341,8 +378,15 @@ export async function setFailedOrder(order: db.Order, error: MarketplaceError, f
 	order.setStatus("failed");
 	order.currentStatusDate = failureDate || order.currentStatusDate;
 	order.error = error.toJson();
-	const user = await User.findOneById(order.userId);
-	metrics.orderFailed(order, user);
+
+	if (order.origin === "p2p") {
+		// TODO: which user to send? maybe both?
```

Comment:
actually both is a good idea. because sender might fail due to timeouts, and recipient might fail because account wasn't created for example. This is just for the error message that will be printed into datadog